src/modules/submit.ts

```diff
@@ -97,3 +102,11 @@ export const submitApp = async (appToSubmit?: string) => {
     handleSubmitAppError(e)
   }
 }
+
+function isFirstPartyApp(runningAccount: string, appVendor: string) {
+  return (
+    (runningAccount === APP_STORE_ACCOUNT ||
+      runningAccount.includes(VTEX_VENDOR)) &&
```

Comment:
Just to make sure: if the `vtex` substring is found anwyhere in the `runningAccount` should we consider it a VTEX account?

If all known VTEX accounts starts with `vtex` I would change the `.includes(` to `.startsWith(`

```suggestion
      runningAccount.startsWith(VTEX_VENDOR)) &&
```

Reply:
They always start with `vtex...`, so these will always render the same effect. I'll change to make it clearer

Reply:
`runningAccount.includes(VTEX_VENDOR))` will evaluate to `true` if `runningAccount = alovtexbf`, but `runningAccount.startsWith(VTEX_VENDOR))` will evaluate to `false`.